jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -41,10 +41,42 @@ class AttentionLayout(Enum):
   BTNH = 0
   BNTH = 1
 
+class MaskType(Enum):
+  NO_MASK = 0
```

Comment:
OOC why not use `None` instead, when a mask is not specified?

Reply:
Just a choice to make it more explicit